src/core/gotolocatorfilter.cpp

```diff
@@ -145,7 +145,7 @@ void GotoLocatorFilter::triggerResult( const QgsLocatorResult &result )
   triggerResultFromAction( result, Normal );
 }
 
-void GotoLocatorFilter::triggerResultFromAction( const QgsLocatorResult &result, const int actionId )
+void GotoLocatorFilter::triggerResultFromAction( const QgsLocatorResult &result, const int )
```

Comment:
Why you do not use `Q_UNUSED` here?
Just wondering. Your way is correct of course.

Reply:
I have no strong feelings either way. Is there an established practice within the QField src tree here?

Reply:
Me neither. I mostly use `Q_UNUSED` since I thought it's the best practice of QGIS but I think it's not defined.
We could define best practices for QField one day, but until we have it written down somewhere it's hard to control.

So this PR is good to merge for me like it is as well. Thanks.